src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Windows.Forms;
 using System.Xml;
+using NuGet;
```

Comment:
In fact, hang on! There ought not to be ANY changes to that file now! Just a sec...